File: dayThree/JAVA/afternoonSession/MiniParkingLot.java
import java.util.Scanner;

public class MiniParkingLot {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        int[] parking = new int[20];
        int user_choice;
        
            System.out.println("\n===== WELCOME TO SEMICOLON MINI PARKING LOT  =====");
            System.out.println(" CHOOSE FROM THE OPTIONS BELOW");
            System.out.println("1. Park a Car");
            System.out.println("2. Remove a Car");
            System.out.println("3. Display Parking Status");
            System.out.println("4. Exit");
            System.out.print("Enter your choice: ");

            user_choice = input.nextInt();
            
    switch (user_choice) {
            // park cars
            case 1:
            boolean parked = false;
            for (int i = 0; i < parking.length; i++) {
                if (parking[i] == 0) {
                    parking[i] = 1;
            System.out.println("Car parked at slot " + (i + 1));
                parked = true;
                break;
            }
            }
                if (!parked) {
            System.out.println("Parking lot is full!");
            }
            displayStatus(parking);
                break;
            
            
   // Remove a car
            case 2:
            System.out.print("Enter slot number to remove car (1-20): ");
                int slot = input.nextInt();
                    if (slot >= 1 && slot <= 20) {
                        if (parking[slot - 1] == 1) {
                            parking[slot - 1] = 0;
            System.out.println("Car removed from slot " + slot);
            } else {
            System.out.println("Slot is already empty.");
            }
            } else {
            System.out.println("Invalid slot number.");
            }

            displayStatus(parking);
                    break;

            // Display parking status
            case 3:
            displayStatus(parking);
                    break;
            default:
                    System.out.println("Invalid choice.");
            }

            //        break;
    }
    


public static void displayStatus(int[] parking) {
        int filled = 0;
        int available = 0;

        System.out.println("\nParking Slots:");

        for (int i = 0; i < parking.length; i++) {
            System.out.println("Slot " + (i + 1) + ": " + parking[i]);

            if (parking[i] == 1) {
                filled++;
            } else {
            available++;
            }
        }
        System.out.println("\nFilled Spaces: " + filled);
        System.out.println("Available Spaces: " + available);
    }
     }

File: dayThree/PYTHON/afternoonSession/mini_parking_lot.py
print("\nParking Slots:")
for i in range(20):
    print(f"Slot {i + 1}: {parking[i]}")

    print(f"\nFilled Spaces: {filled}")
    print(f"Available Spaces: {available}")


while True:
    print("\n===== PARKING LOT SYSTEM =====")
    print("1. Park a Car")
    print("2. Remove a Car")
    print("3. Display Parking Status")
    print("4. Exit")

    user_choice = int(input("Enter your choice: "))

# Park a car
    if choice == 1:
        parked = False

        for i in range(20):
            if parking[i] == 0:
                parking[i] = 1
                print(f"Car parked at slot {i + 1}")
                parked = True
                break

        if not parked:
            print("Parking lot is full!")

        display_status()

# Remove a car
    elif choice == 2:
        slot = int(input("Enter slot number to remove car (1-20): "))

        if 1 <= slot <= 20:
            if parking[slot - 1] == 1:
                parking[slot - 1] = 0
                print(f"Car removed from slot {slot}")
            else:
                print("Slot is already empty.")
        else:
            print("Invalid slot number.")

        display_status()

# Display status
    elif choice == 3:
        display_status()

# Exit
    elif choice == 4:
        print("Exiting program...")
        break

    else:
        print("Invalid choice.")
